lib/util.js

```diff
@@ -151,7 +152,26 @@ module.exports.readFilesToArray = function(fileList) {
   var contentArray = [];
   fileList.forEach(function(filePath) {
     try {
-      contentArray.push(fs.readFileSync(filePath));
+      let extension = filePath.split('.').pop();
+      let content = fs.readFileSync(filePath);
+      if(extension == 'p12' || extension == 'pfx'){
+        let p12Der = forge.util.decode64(content.toString('base64'));
+        let p12Asn1 = forge.asn1.fromDer(p12Der);
+        let p12;
+        try{
+          p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, process.env.KEYSTORE_PASSWORD || "");
+        } catch (e) {
+          loggers.bootstrapLogger.warn("ZWED0173W", e.message);
+          p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, "");
+        }
+        let certData = p12.getBags({bagType: forge.pki.oids.certBag});
```

Comment:
p12 files can be like an archive of multiple types of security objects (keys, certificates, crls etc)
the function this is in is generic: the output is assigned to either keys, certs, cert authorities, depending on which section of the processing we are in.

but, this new code appears to assume its only being used for certs, which is not true.